crates/egui/src/widgets/plot/mod.rs

```diff
@@ -35,9 +32,14 @@ type AxisFormatter = Option<Box<AxisFormatterFn>>;
 type GridSpacerFn = dyn Fn(GridInput) -> Vec<GridMark>;
 type GridSpacer = Box<GridSpacerFn>;
 
+type CoordinatesFormatterFn = dyn Fn(&PlotPoint, &PlotBounds) -> String;
+
+const LINKED_PLOT_AXES_KEY: &str = "egui_linked_plot_axes";
+const LINKED_PLOT_CURSORS_KEY: &str = "egui_linked_plot_cursors";
```

Comment:
Not sure what the best approach is to add something to the egui memory.

Reply:
If data is to only be identified by type, use `Id::null`: https://docs.rs/egui/latest/egui/util/id_type_map/struct.IdTypeMap.html